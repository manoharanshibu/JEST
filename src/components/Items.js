import React, { Component } from 'react';

export default class Items extends Component {
    render(){
        const { items = [] } = this.props;
        return (
            <ul>
              {items.map(item => <li key={item}>{item}</li>)}
            </ul>
          );
    }
}