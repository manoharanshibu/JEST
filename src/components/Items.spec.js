import React from 'react';
import renderer from 'react-test-renderer';

import Items from './Items';

describe('snap shot test', () => {
  it('renders correctly when there are no items', () => {
    const tree = renderer.create(<Items />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});