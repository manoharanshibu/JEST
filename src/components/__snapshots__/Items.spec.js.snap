// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snap shot test renders correctly when there are no items 1`] = `<ul />`;
